CONTRIBUTING.md

```diff
@@ -26,7 +26,7 @@ Before making a contribution, it is important to make sure that the change you w
         * Features are forever, always think about whether a new feature really belongs to the core framework or if it should be implemented as a module
         * Code must conform to standard style guidelines and pass all tests
     6. New files must:
-        * Have a Lightbend copyright header in the style of ``Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>``.
+        * Have a Lightbend copyright header in the style of ``Copyright (C) 2017 Lightbend Inc. <https://www.lightbend.com>``.
```

Comment:
IIUC ranges should be used. See also https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/Main.scala